modules/bmo-getting-the-hostfirmwaresettings-resource.adoc

```diff
@@ -0,0 +1,33 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+[id="getting-the-hostfirmwaresettings-resource_{context}"]
+= Getting the HostFirmwareSettings resource
+
+The `HostFirmwareSettings` resource contains the vendor-specific BIOS properties of a physical host. You must get the `HostFirmwareSettings` resource for a physical host to review its BIOS properties.
+
+.Procedure
+
+
+. (Optional) Get the list of hosts:
++
+[source,terminal]
+----
+$ oc get hfs -n openshift-machine-api
+----
+
+
+. Get the `HostFirmwareSettings` resource:
++
+[source,terminal]
+----
+$ oc get hfs -n openshift-machine-api -o yaml
+----
++
+or
++
+[source,terminal]
+----
+$ oc get hostfirmwaresettings -n openshift-machine-api -o yaml
```

Comment:
This is the same command as the previous one (oc get hfs -n openshift-machine-api -o yaml), should just show this once.

Reply:
IMHO, it's ok to show short and long object name as it was dome for BMH

But what is missing here is 
(Optional) Get the HostFirmwareSettings resource for a specific host:

$ oc get hfs -n openshift-machine-api/<host_name> -o yaml
Where <host_name> is the name of the host.